Automattic/sensei
# Add course theme theme

Closes  #4446

### Changes proposed in this Pull Request

* If course theme is active for a lesson, redirect to a `/learn/lesson/title` page
* If on that page, use a bundled theme instead of the active theme
* Add bundled theme, move current templates into it

### Testing instructions

* Enable the feature flag: `add_filter( 'sensei_feature_flag_{THE_FLAG_NAME}', '__return_true' );`
* ~Flush rewrite rules: open `/wp-admin/options-permalink.php` and click `Save changes`~ 
* Enable course theme for a course
* Open a lesson of that course as a student
* Page should redirect to `/learn/lesson/title`
* Course theme should show up
* Lesson content should be mostly unstyled. (Default block library CSS is still loaded)


### Screenshot / Video
